test/common/config/config_provider_impl_test.cc

```diff
@@ -20,6 +19,22 @@ using testing::InSequence;
 
 class DummyConfigProviderManager;
 
+class DummyConfig : public Envoy::Config::ConfigProvider::Config {
+public:
+  DummyConfig() {}
```

Comment:
not needed

Reply:
it's used in the test below. see around make_shared<DummyConfig>();